File: week-02/day-02/double_items.py
# - Create an array variable named `ag`
#   with the following content: `[3, 4, 5, 6, 7]`
# - Double all the values in the array

ag = [3, 4, 5, 6, 7]

def double(input_list):
    for i in range(len(input_list)):
        input_list[i] *= 2

double(ag)
print(ag)

File: week-07/day-02/double-items.js
'use strict';
// - Create an array variable named `ag` with the following content: `['Gin', 'Whiskey', 'Wine', 'Beer']`
// - Double all the strings in the array, use a built in array method instead of a loop
// It should print: ['GinGin', 'WhiskeyWhiskey', 'WineWine', 'BeerBeer']`

const ag = ['Gin', 'Whiskey', 'Wine', 'Beer'];

const agDouble = ag.map(string => string.repeat(2));
console.log(agDouble);